### PR TITLE
remove references to manually pushed charts

### PR DESCRIPTION
**Pull request by fgimenez**

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -17,25 +17,16 @@ directory.
...
-                   -n cert-resource-lab \
-                   --set clusterName=my-cluster \
-                   --set commonDomain=my-common-domain
+helm install -n cert-operator-lab ./cert-operator-lab-chart/ --set imageTag latest --wait
 ```
 
-- Note: `clusterName` and `commonDomain` chart values must match the values used
-  during this guide.
-
 [helm registry plugin]: https://github.com/app-registry/appr-helm-plugin
```
Comment (kopiczko) [inline examples/README.md]: This is not needed anymore.

File: examples/README.md [hunk 2]
```diff
@@ -17,25 +17,16 @@ directory.
...
-                   -n cert-operator-lab \
-                   --set imageTag=latest \
-                   --set clusterName=my-cluster \
-                   --set commonDomain=my-common-domain \
-                   --wait
-helm registry install quay.io/giantswarm/cert-resource-lab-chart -- \
-                   -n cert-resource-lab \
-                   --set clusterName=my-cluster \
-                   --set commonDomain=my-common-domain
+helm install -n cert-operator-lab ./cert-operator-lab-chart/ --set imageTag latest --wait
```
Comment (kopiczko) [inline examples/README.md]: I'd replace it with:

```
git clone github.com/giantswarm/cert-operator ./cert-operator

helm \
    install -n cert-operator-lab ./cert-operator/examples/cert-operator-lab-chart/ \
    --set clusterName=my-cluster \
    --set commonDomain=my-common-domain \
    --set imageTag latest \
    --wait
```
WDYT?

Comment (fgimenez) [reply examples/README.md]: Much better :) I've also added a note about the waiting for the TPR to be ready before trying to install the resources chart, pls take a look.

File: examples/README.md (hunk 1 above)
Comment (fgimenez) [reply examples/README.md]: Done

**Pull request merged**